servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -1925,6 +2278,16 @@
             "x-ms-skip-url-encoding": true
           },
           {
+            "name": "ForceRemove",
+            "in": "query",
+            "description": "The force remove",
```

Comment:
Please provide better descriptions. Automated descriptions dont help the customer. REST API documentation is autogenerated from the spec and published to docs.microsoft.com website where every one can see.